CMakeLists.txt

```diff
@@ -489,57 +486,24 @@ if(UNIX)
     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")
 endif(UNIX)
 
-    if(UNIX)
-
-        if(ANTILIB_PATH)
-            ADD_DEFINITIONS(-DANTILIB_PATH="${ANTILIB_PATH}")
-
-            # Generally package manager takes care of this while installing the new library, but not always
-            # especially with cmake
-            ADD_DEFINITIONS(-DLD_LIBRARY_PATH="${ANTILIB_PATH}:${LD_LIBRARY_PATH}")
-
-            # those instructions have to be before add_executable() or add_library() is called, otherwise it has no effect
-            set(CMAKE_INSTALL_RPATH "${ANTILIB_PATH}")
-            set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-        endif()
-
-            add_library( antilib
-                SHARED
```

Comment:
I have a suspicion that `xcb` linking was not needed because this used to be compiled as a shared library, it was only linked dynamically, and `xcb` is a requirement for `qt`. @pktiuk is this a wild guess or would `xcb` be needed at linking, too?

Reply:
We directly need xcb because some functions fro xcb/xcb.h where used.
And these are not a part of qt in any ways..